### PR TITLE
v6.2 Boards

### PR DESCRIPTION
**Pull request by chenilim**
Add a notice to highlight Boards.

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -129,5 +129,23 @@
...
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">=6.2"],
+      "displayDate": ">= 2021-12-10T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Manage your projects and tasks in one place",
+        "description": "View your active projects and work items, update weekly status and more by selecting Boards from the top-left menu.",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/focalboard.gif",
```
Comment (chenilim) [inline notices.json:145]: @asaadmahmood, please create a new GIF to use for the new notice. Thanks. cc @wuwinson.

File: notices.json [hunk 2]
```diff
@@ -129,5 +129,23 @@
...
+      "serverVersion": [">=6.2"],
+      "displayDate": ">= 2021-12-10T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Manage your projects and tasks in one place",
+        "description": "View your active projects and work items, update weekly status and more by selecting Boards from the top-left menu.",
+        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Boards_v6.2.gif",
+        "actionText": "Open Boards",
+        "actionParam": "/boards/welcome"
```
Comment (esethna) [inline notices.json:147]: We'll need to make sure this actually works cc// @ogi-m

File: notices.json [hunk 3]
```diff
@@ -129,5 +129,23 @@
...
       }
     }
+  },
+  {
+    "id": "v6.2_boards",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">=6.2"],
+      "displayDate": ">= 2021-12-10T00:00:00Z"
```
Comment (esethna) [inline notices.json:139]: Not needed if the date is in the past

**Review by esethna — APPROVED** [review]

**Comment by ogi-m** [comment]
@chenilim the link works! Just to confirm - we want to show this on cloud as well?

**Comment by chenilim** [comment]
Yes, we want to show this on Cloud as well. I'm out from tomorrow. Can you help get this merged? Thanks @ogi-m! cc @wuwinson.

**Review by ogi-m — APPROVED** [review]

**Comment by ogi-m** [comment]
Thanks Chen! @esethna can you please merge this?

**Comment by esethna** [comment]
@ogi-m I thought this won't appear on cloud if the `serverversion` condition is used?

**Comment by ogi-m** [comment]
@esethna It was a long time ago but if I remember correctly, it did work the way you described it but it was actually an issue since we couldn't target both cloud and onprem if there was a server version specified (since cloud wouldn't meet the requirement). Then we changed it so that the version check ignores cloud.

**Comment by ogi-m** [comment]
So if we want to exclude cloud we need to specify it with the _InstanceType_ condition

**Comment by esethna** [comment]
Got it, thanks @ogi-m. I'll update the read me

**Review by esethna — APPROVED** [review]

**Pull request merged**